themes/api-platform/layouts/_default/_markup/render-codeblock.html

```diff
@@ -0,0 +1,17 @@
+{{ $result := transform.HighlightCodeBlock . }}
+{{ replaceRE `\x60&lt;a href=&#34;(.+)?&#34;&gt;(.+)?&lt;\/a&gt;\x60` "<a href=\"$1\">$2</a>" $result.Wrapped | safeHTML }}
```

Comment:
Why this replacement?

Reply:
There are links in code blocks?

Reply:
Could you import the code of the default markup in yours to avoid duplication?

Reply:
> There are links in code blocks?

This code is written so I think so, can you confirm this @soyuka, please?

Reply:
> Could you import the code of the default markup in yours to avoid duplication?

We should create a partial to do this and use it in both places. I will do it if you wish.

Reply:
It could be nice 🙂

Reply:
Done @alanpoulain